integreatly/integreatly-1.5.8/integreatly-operator.1.5.8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,336 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: integreatlyoperator.1.5.8
+  namespace: placeholder
+  annotations:
+    alm-examples: "{}"
+    categories: Integreating greatly
+    certified: 'false'
+    containerImage: quay.io/integreatly/integreatly-operator:v1.5.8
+    createdAt: 2019-04-30 12:39:00
+    support: Integreatly
+    description: Greatly integrating integreatly greatness
+
+spec:
+  displayName: Integreatly
+  description: Integreatly installation and reconciliation operator
+  keywords: ['integreatly']
+  version: 1.5.8
+  maturity: alpha
+  replaces: integreatlyoperator.1.5.7
```

Comment:
@matskiv a little confused , taught I had that done here , is there somewhere else I need to change it ??

Reply:
@austincunningham You are right. Sorry for the the confusion. I looked just on keycloak-operator version bump, and it needs to be handled the same way as you have done with integreatly-operator.

Reply:
So keycloak-operator needs previous versions directory as well like so ?
![image](https://user-images.githubusercontent.com/16667688/63159995-9436a980-c014-11e9-9e72-0dd956cd938b.png)

Reply:
yes :)